
---
float32 origin_x
float32 origin_y
float32 resolution
sensor_msgs/Image height_map
